geoengine/types.py

```diff
@@ -4,26 +4,26 @@
 from datetime import datetime
 
 
-class Bbox:
+class QueryRectangle:
```

Comment:
do we not want to distinguish between raster and vector query rectangles?

Reply:
Currently, I see no need in doing so. I guess this will become important once we do something with these rectangles in Python. So we should keep it in mind.